fhir-server/src/main/java/com/ibm/fhir/server/util/FHIRRestHelper.java

```diff
@@ -2949,18 +2951,32 @@ public void validateInteraction(Interaction interaction, String resourceType) th
     }
 
     @Override
-    public Bundle doHistory(MultivaluedMap<String, String> queryParameters, String requestUri) throws Exception {
+    public Bundle doHistory(MultivaluedMap<String, String> queryParameters, String requestUri, String resourceType) throws Exception {
         log.entering(this.getClass().getName(), "doHistory");
 
         // Validate that the interaction is allowed
-        validateInteraction(Interaction.HISTORY, "Resource");
+        if (resourceType == null) {
+            validateInteraction(Interaction.HISTORY, "Resource");
+        } else {
+            validateInteraction(Interaction.HISTORY, resourceType);
+        }
 
         // extract the query parameters
         FHIRRequestContext requestContext = FHIRRequestContext.get();
+        
+        if (requestContext.isReturnPreferenceDefault()) {
+            // For _history, override the default preference to be REPRESENTATION so resources
+            // are returned in the response bundle per the R4 spec.
+            requestContext.setReturnPreference(HTTPReturnPreference.REPRESENTATION);
+        }
         FHIRSystemHistoryContext historyContext =
                 FHIRPersistenceUtil.parseSystemHistoryParameters(queryParameters, HTTPHandlingPreference.LENIENT.equals(requestContext.getHandlingPreference()));
 
+        // If HTTPReturnRepresentation is REPRESENTATION, we fetch the resources and include them
```

Comment:
```suggestion
        // If HTTPReturnPreference is REPRESENTATION, we fetch the resources and include them
```

Reply:
I knew I'd make that mistake somewhere :-)

Reply:
fixed